src/main/java/com/data/dataxer/models/domain/PriceOffer.java

```diff
@@ -22,8 +22,8 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @OneToMany(mappedBy = "priceOffer", orphanRemoval = true, cascade = CascadeType.ALL, fetch = FetchType.LAZY)
-    List<PriceOfferPack> packs = new ArrayList<>();
+    @OneToMany(mappedBy = "documentId", orphanRemoval = true, cascade = CascadeType.ALL, fetch = FetchType.LAZY )
+    List<Document> packs = new ArrayList<>();
```

Comment:
tomuto moc nerozumiem, preco je to List documentov ? nemalo by to byt skor list documentPacks ?